components/user-menu.scss

```diff
@@ -0,0 +1,83 @@
+$header-img-size: 150px;
+$menu-width: 250px;
+
+.hdd-user-menu {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+
+  height: auto;
+  width: $menu-width;
+  border-radius: 4px;
+  background: var(--hdd-color-white);
+  box-shadow: $shadow-strong;
+  padding: var(--hdd-spacing-3);
+
+  &_image {
+    width: $header-img-size;
+    height: $header-img-size;
+    margin: var(--hdd-spacing-1) 0;
+    border-radius: 100%;
+
+    background: var(--hdd-color-grey-light);
+    background-position: center;
+  }
+
+  &_initials {
+    margin: var(--hdd-spacing-1) 0;
+    background-color: var(--hdd-color-blue);
+    width: $header-img-size;
+    height: $header-img-size;
+    margin-right: var(--hdd-spacing-1);
+    border-radius: 100%;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    color: var(--hdd-color-white);
+    text-transform: uppercase;
+    font-size: 2rem;
+  }
+
+  &_name {
+    margin: var(--hdd-spacing-1) 0;
+    font-size: 1.25rem;
+    letter-spacing: 0.15px;
+    color: var(--hdd-color-black);
+    text-align: center;
+  }
+
+  &_email {
+    margin-top: var(--hdd-spacing-1);
+    margin-bottom: var(--hdd-spacing-2);
+    letter-spacing: 0.15px;
+    font-size: 1rem;
+    text-align: center;
+    color: var(--hdd-color-black);
```

Comment:
Vi har en egen (--hdd-color-font-color) for tekst, kan brukes på emailen her, og evt ha navnet i ren svart om den skal navnet skal fremheves litt.

Reply:
Anna ville ha denne fargen, men skal sjekke med hun